kotti/util.py

```diff
@@ -331,7 +331,13 @@ def disambiguate_name(name):
     return u'-'.join(parts)
 
 
-def title_to_name(title, blacklist=()):
+def title_to_name(title, blacklist=(), max_length=None):
+    """ If max_length is None, fallback to the ``name`` column
+        size (:class:`kotti.resources.Node`)
+    """
+    if max_length is None:
+        from kotti.resources import Node
+        max_length = Node.name.property.columns[0].type.length
```

Comment:
If we made this ``max_length = Node.name.property.columns[0].type.length - 10``, it would also address #31.  Tests would of course need to be adjusted accordingly.